specification.md

```diff
@@ -278,7 +274,7 @@ customary in the language in which the library is implemented
 
 #### Interaction with LRMs and Scalability
 
-Implementations should use bulk status operations when interacting with
+Implementations must use bulk status operations when interacting with
```

Comment:
Are we sure we want "must" instead of "should" here>

Reply:
I am very much in favor of `should` - thus the original edit.  I'll do a separate PR for that one if that's ok for you...

Reply:
A separate PR sounds like a lot of work. I'm fine with this being kept here.